php/Terminus/Loggers/Logger.php

```diff
@@ -55,13 +55,25 @@ public function __construct(
   }
 
   /**
-    * Logs with an arbitrary level.
-    *
-    * @param [mixed]  $level
-    * @param [string] $message
-    * @param [array]  $context
-    * @return [void]
-    */
+   * Logs an error message and then exits with -1
+   *
+   * @param [string] $message Message to convey to user
+   * @param [array]  $context Parts of message to interpolate
+   * @return [void]
+   */
+  public function error($message, array $context = array()) {
+    parent::error($message, $context);
+    exit(-1);
```

Comment:
I'd like a second opinion on this implementation. It seems like an excellent idea to me, but is there a reason why we'd want to log something of severity ERROR but not exit?